pkg/ignition/ignition.go

```diff
@@ -52,7 +52,7 @@ func New(c client.Client) (*Ignition, error) {
 		return nil, err
 	}
 
-	configuration, report, err := ignCfg.Parse(renderedWorker.Spec.Config.Raw)
+	configuration, report, err := ignCfg.ParseCompatibleVersion(renderedWorker.Spec.Config.Raw)
```

Comment:
This will parse ignition versions less or equal to 3.4 which makes the log message in [line#62](https://github.com/openshift/windows-machine-config-operator/pull/1743/files#diff-5f8bc59c6ca4160f53648587fbed538adbd305ad0dfb837bd3124f2ee84fde9bR62) inaccurate as it will always log the resulting version not the source one. Consider adding a log showing the actual ignition version in the `renderedWorker`


```go
	log.V(1).Info("parsed", "machineconfig", renderedWorker.GetName(), "ignition version",
		configuration.Ignition.Version)
```

Reply:
@jrvaldes I'm not sure how you get the actual version from the renderedWorker like you are suggestion, can you help me out here?

Reply:
@sebsoto adding a log message with the ignition version present in the renderedWorker adds another unmarshaling to the execution flow. I recommend just reword the log message in [line#62](https://github.com/openshift/windows-machine-config-operator/pull/1743/files#diff-5f8bc59c6ca4160f53648587fbed538adbd305ad0dfb837bd3124f2ee84fde9bR62) for example:

```go
	log.V(1).Info("processing", "machineconfig", renderedWorker.GetName(), "ignition version",
		configuration.Ignition.Version)
```